dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/test/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/PodLogsTest.java

```diff
@@ -33,7 +33,8 @@ public void shouldHandlePodNotFound() throws Exception {
         PodLogs command = createCommand();
         command.name = "mickey-mouse";
         command.maxWaitAttempts = 2; // total timeout of 4 seconds
-        command.doCall();
+        int exit = command.doCall();
+        Assertions.assertEquals(1, exit);
```

Comment:
Just wonder why exit code 1 is expected, is the command supposed to fail somehow ? If so maybe add a code comment to tell this reason so we wont get confused later looking back at this code

Reply:
It is debatable, wether the `jbang camel kubernetes logs ...` command should "fail" when the Pod is not there. User feedback (in print the output) is clear. We might as well succeed the command and reserve failure to unexpected server failures. Would that be better?

Reply:
Yes I think this is better

Reply:
Done